docs/en/concepts-and-designs/mal.md

```diff
@@ -57,6 +57,25 @@ For example, this filters all instance_trace_count samples for values >= 33:
 ```
 instance_trace_count.valueGreaterEqual(33)
 ```
+### K8s
+MAL support add specific Labels to the samples that collect from K8s metrics collectors.
+This feature need OAP Server has the authority to access the K8s's `API Server`.
+
+#### k8sTagServiceByPodName
+`k8sTagServiceByPodName(podName,serviceName)`. Add a sevice Label to the sample that already has a pod name.
```

Comment:
I think base on your description, this should be renamed to 
```suggestion
`relabelByK8sMeta(newLabelName, type, existingLabelName)`. Add a new label to the sample family based on an existing label's value. Provide several internal converting types, including
- Pod2Service
```

Reply:
Does this look better?

Reply:
done